src/IIS/Settings/Bindings/FluentAPI/IBindings.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Cake.IIS.Settings.Bindings.FluentAPI
+{
+    public interface IBindings
```

Comment:
Interface exposed Fluent API only. It will be removed with extension methods implementation.